include/core/ib_particle.h

```diff
@@ -32,14 +32,13 @@
 
 using namespace dealii;
 
+
```

Comment:
`/**
  * @brief This class defines values related to a particle used in the sharp interface IB. These values are used in the
  * solver
  * 
  * @tparam dim  An integer that denotes the number of spatial dimensions
  */